lib/admin/WP_Auth0_Admin_Appearance.php

```diff
@@ -472,19 +468,18 @@ public function render_custom_signup_fields( $args = array() ) {
 	}
 
 	public function basic_validation( $old_options, $input ) {
-		$input['form_title']         = sanitize_text_field( $input['form_title'] );
-		$input['icon_url']           = esc_url( $input['icon_url'], array( 'http', 'https' ) );
-		$input['social_big_buttons'] = ( isset( $input['social_big_buttons'] ) ? $input['social_big_buttons'] : 0 );
-		$input['gravatar']           = ( isset( $input['gravatar'] ) ? $input['gravatar'] : 0 );
-		$input['language']           = sanitize_text_field( $input['language'] );
-		$input['primary_color']      = sanitize_text_field( $input['primary_color'] );
+		$input['form_title']    = empty( $input['form_title'] ) ? '' : sanitize_text_field( $input['form_title'] );
+		$input['icon_url']      = empty( $input['icon_url'] ) ? '' : esc_url( $input['icon_url'], array( 'http', 'https' ) );
+		$input['gravatar']      = isset( $input['gravatar'] ) ? $input['gravatar'] : 0;
```

Comment:
I know you didn't change the logic around here but should the gravatar have a sanitize or escape?

Reply:
It's checked as truth-y. Any value == `true`

Reply:
More specifically ... any value is stored, then a truth-y value means "use the incoming Auth0 avatar"